src/main/java/org/sagebionetworks/web/client/presenter/QuizPresenter.java

```diff
@@ -1,379 +1,33 @@
 package org.sagebionetworks.web.client.presenter;
 
```

Comment:
A shell of a class that it used to be.  Now it just loads the SRC implementation.